Add ability for custom modal titles and button labels (#52)
saade/filament-fullcalendar: README.md

}
```

> You can override the `getCreateModalTitle()` method to change the modal title to a custom one:

Reviewer: ```suggestion
You can override the `getCreateEventModalTitle()` method to change the modal title to a custom one:
```

```php

> public function getCreateModalTitle(): string 

Reviewer: ```suggestion
public function getCreateEventModalTitle(): string 
```

{
    return __('filament::resources/pages/create-record.title', ['label' => $this->getModalLabel()]);
}
```

> You can override the `getCreateModalSubmitButtonLabel()` and `getCreateModalCloseButtonLabel()` methods to change the modal button labels to custom labels:

Reviewer: ```suggestion
You can override the `getCreateEventModalSubmitButtonLabel()` and `getCreateEventModalCloseButtonLabel()` methods to change the modal button labels to custom labels:
```

```php

> public function getCreateModalSubmitButtonLabel(): string 

Reviewer: ```suggestion
public function getCreateEventModalSubmitButtonLabel(): string 
```

{
    return __('filament::resources/pages/create-record.form.actions.create.label');
}

> public function getCreateModalCloseButtonLabel(): string 

Reviewer: ```suggestion
public function getCreateEventModalCloseButtonLabel(): string 
```

}
```

> You can override the `getEditModalTitle()` method to change the modal title to a custom one:

Reviewer: ```suggestion
You can override the `getEditEventModalTitle()` method to change the modal title to a custom one:
```

```php

> public function getCreateModalTitle(): string 

Reviewer: ```suggestion
public function getCreateEventModalTitle(): string 
```

{
    return __('filament::resources/pages/create-record.title', ['label' => $this->getModalLabel()]);
}
```

> You can override the `getEditModalSubmitButtonLabel()` and `getEditModalCloseButtonLabel()` methods to change the modal button labels to custom labels:

Reviewer: ```suggestion
You can override the `getEditEventModalSubmitButtonLabel()` and `getEditEventModalCloseButtonLabel()` methods to change the modal button labels to custom labels:
```

```php

> public function getEditModalSubmitButtonLabel(): string 

Reviewer: ```suggestion
public function getEditEventModalSubmitButtonLabel(): string 
```

{
    return __('filament::resources/pages/edit-record.form.actions.save.label');
}

> public function getEditModalCloseButtonLabel(): string 

Reviewer: ```suggestion
public function getEditEventModalCloseButtonLabel(): string 
```
